pure_utils/bigquery.go

```diff
@@ -0,0 +1,23 @@
+package pure_utils
+
+import "cloud.google.com/go/bigquery"
+
+func NullStringFromPtr(ptr *string) bigquery.NullString {
```

Comment:
Could we specify a name that signifies this is specifically for BigQuery? Like `BigQueryNullStringFrom`?

Reply:
They are, for BQ's `OPTIONAL` columns.